src/dht/client.rs

```diff
@@ -652,7 +758,11 @@ impl Client {
         }
     }
 
-    //Need for testing from console.
+    /// Handles an input line by parsing and executing the corresponding command.
```

Comment:
The formatting here is a bit inconsistent, usually there's an empty line before parameters